db-async-common/src/main/java/com/github/jasync/sql/db/StaticCredentialsProvider.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.jasync.sql.db
+
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.CompletionStage
+
+class StaticCredentialsProvider(val username: String, val password: String?) : CredentialsProvider {
```

Comment:
kotlin allows multiple classes in the same file, so this fits CredentialsProvider.kt